buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/TestClustersPlugin.java

```diff
@@ -120,10 +105,18 @@ public void apply(Project project) {
                 sync.from((Callable<List<FileTree>>) () ->
                     helperConfiguration.getFiles()
                         .stream()
-                        .map(project::zipTree)
+                        .map(file -> {
+                          if (file.getName().endsWith(".zip")) {
+                              return project.zipTree(file);
+                          } else if (file.getName().endsWith("tar.gz")) {
+                              return project.tarTree(project.getResources().gzip(file));
```

Comment:
We can actually just call `tarTree` directly here as it will interpret `.tar.gz` as a compressed file automatically.

> Unless custom implementation of resources is passed, the tar tree attempts to guess the compression based on the file extension.

Reply:
done